.gitignore

```diff
@@ -65,6 +66,7 @@ dist/# See https://help.github.com/ignore-files/ for more about ignoring files.
 
 # dependencies
 /node_modules
+# semantic.json
```

Comment:
Why does the changelog say the `semantic.json` is deleted but you're including it still in the repo? So what exactly is happening with this file?

Reply:
Updated the changelog - we need `semantic.json` to get the autoInstall

Reply:
I originally deleted semantic.json to enable the interactive installer, but Greg noticed afterward that we can enable autoInstall in this file, so we actually need to keep it.

Reply:
👍